.gitlab-ci.yml

```diff
@@ -59,8 +60,18 @@ deploy:
   image: "juliagpu/julia:v1.3-${CI_IMAGE_TAG}"
 
   before_script:
-      - apt-get -qq update
-      - apt-get install -y cmake xorg-dev mesa-utils p7zip-full
+    - apt-get -qq update
+    - apt-get install -y cmake xorg-dev mesa-utils p7zip-full
+    - 'which ssh-agent || ( apt-get update -y && apt-get install openssh-client -y )'
+    - eval $(ssh-agent -s)
+    - ssh-add $(echo $GIT_SSH_KEY)
+    - git config --local user.email "ci@gitlab.com"
+    - git config --local user.name "Gitlab CI"
+    - git clone --depth=1 https://github.com/JuliaPlots/MakieReferenceImages
 
   script:
     - julia --project test/upgrade_recordings.jl
+    - cd MakieReferenceImages
```

Comment:
is this upgrading reference images after every test run?

Reply:
Nope...only when you manually run the deploy stage.  There's a button to do that in the Gitlab UI.

Reply:
Oh sweeet :) And you can browse the artifacts?

Reply:
Yep, the artifacts from the test stage are available for 3 days.  You can also press a button to keep them for longer.